README.md

```diff
@@ -524,6 +524,7 @@ When learning CS there are some useful sites you must know to get always informe
 - [IT Enthusiast](http://rodiongork.tumblr.com) : IT Enthusiast
 - [Joel on Software](https://www.joelonsoftware.com) : The blog of the CEO of StackOverflow
 - [Late Developer](https://latedev.wordpress.com) :  Random thoughts of an old C++ guy
+- [1ucasvb's laboriginal math and physics visualization](http://1ucasvb.tumblr.com/) : Lucas Vieira Barbosa's lab original math and physics visualization
```

Comment:
@srinivasreddy please remove the trailing forward slash at the end of this URL `http://1ucasvb.tumblr.com/`. It's the reason why Travis is failling. [https://travis-ci.org/sdmg15/Best-websites-a-programmer-should-visit/builds/269888298?utm_source=github_status&utm_medium=notification](`Awesome-lint` doesn't like it.)
```
0.69s$ awesome-lint
  69:3-69:50    error    Remove trailing slash (http://highscalability.com)  trailing-slash
  527:3-527:85  error    Remove trailing slash (http://1ucasvb.tumblr.com)   trailing-slash
✖ 2 errors
```